ai_diffusion/workflow.py

```diff
@@ -153,6 +155,29 @@ def load_checkpoint_with_lora(w: ComfyWorkflow, checkpoint: CheckpointInput, mod
     if arch.supports_attention_guidance and checkpoint.self_attention_guidance:
         model = w.apply_self_attention_guidance(model)
 
+    # Apply MagCache as a model patch if enabled  
+    if checkpoint.magcache_enabled:
+        if arch in [Arch.flux, Arch.flux_k]:
+            print(f"Applying MagCache patch for {arch}")
+            
+            model_type = "flux_kontext" if arch is Arch.flux_k else "flux"
+            
+            try:
```

Comment:
no need to wrap in try/except, this should not fail (or do so loudly if the code is broken)